packages/smart-wallet/src/smartWallet.js

```diff
@@ -124,6 +124,7 @@ const mapToRecord = map => Object.fromEntries(map.entries());
  *   purseBalances: MapStore<RemotePurse, Amount>,
  *   updatePublishKit: PublishKit<UpdateRecord>,
  *   currentPublishKit: PublishKit<CurrentWalletRecord>,
+ *   possiblyExitableOffers: MapStore<import('./offers.js').OfferId, import('./offers.js').OfferStatus>,
```

Comment:
@turadg I renamed this because:

1) It makes it easier to understand why we're storing them.
2) I don't think it's the exact same as currently seated offers. If we have an error, the offer could still be seated, but at that point the smart wallet should stop storing it and try to unseat it automatically.

Reply:
After my latest changes, point 2 is irrelevant though. We need to still store it in case the exit fails due to a deadline or something.

Reply:
I forget what the earlier term was but this one feels klunky. let's chat higher bandwidth